src/main/java/ejercicio/banco/repository/MySqlAccountRepository.java

```diff
@@ -57,22 +54,22 @@ public Account find(int id) {
                     Float money = rs.getFloat("money");
                     String iban = rs.getString("iban");
                     Account account = new Account(id, name, money, iban);
-                    closeConections(con);
+                    DataBaseUtil.closeConections(con);
                     return account;
                 }
             }
-            closeConections(con);
         } catch (Exception e) {
             e.printStackTrace();
         }
+        DataBaseUtil.closeConections(con);
```

Comment:
Why do you close the connection twice?

Reply:
Since I was not using finally, I was closing it inside the try and outside of it. Now that I am using finally, I don't need to use it twice.